src/base/task.cc

```diff
@@ -851,7 +855,7 @@ void TaskScheduler::WaitForTerminateCompletion() {
 
     int count = 0;
     int threadsRunning;
-    pid_t pid = getpid();
+    pid_t pid = osspecific_getpid();
```

Comment:
As per our Tuesday discussion, we should avoid breaking compatibility if possible.